package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "lab-irvine",
+  "version": "1.0.0",
+  "description": "401 JS -- Class 01 Lab modular patterns ===",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/irvinemd55/lab-01-modular-patterns.git"
+  },
+  "author": "irvinedowning@gmail.com",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/irvinemd55/lab-01-modular-patterns/issues"
+  },
+  "homepage": "https://github.com/irvinemd55/lab-01-modular-patterns#readme"
+  "devDependencies": {
```

Comment:
You need to put a comma right after "patterns#readme" so that the JSON can be properly parsed. E.g.:

```
"homepage": "https://github.com/irvinemd55/lab-01-modular-patterns#readme"

```
When I tried to run "npm i" on my terminal with your code, I got a JSON parsing error because of this.